steely/plugins/choose.py

```diff
@@ -0,0 +1,18 @@
+'''for those indecisive moments'''
+
+__author__ = 'itsdoddsy'
+COMMAND = 'choose'
+
+import re
+import random
+
+ERRORS = ['fucking moron',
+          'how about no',
+          'pikachu',]
+
+def main(bot, author_id, message, thread_id, thread_type, *args, **kwargs):
+    try:
+        args = re.findall(r"\w+\b(?<!\bchoose)", message)
+        bot.sendMessage(random.choice(args), thread_id=thread_id, thread_type=thread_type)
+    except IndexError:

```

Comment:
Wait so this exception is just for covering your own errors? Can you maybe comment what kind of errors produce these issues?

Reply:
When someone uses `.choose` but doesn't give anything to choose from it raises IndexError